public/css/index.css

```diff
@@ -0,0 +1,175 @@
+body {
+    width: 100%;
+}
+.type {
+    font-family: 'Open Sans', sans-serif;
+}
+.container-fluid .row {
+    margin: auto !important;
+}
+#home-section {
+    background: url('/img/card.jpg') no-repeat;
+    min-height: 700px;
+    background-size: cover;
+    background-attachment: fixed;
+    /* opacity: 0.6; */
+}
+#overlay {
+    background-color:rgba(0,0, 0, 0.7);
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    min-height: 700px;
+}
+#cards {
+    background-color: black;
+    color: #fff;
+    padding:5px 5px;
+    border-radius: 5px; 
+}
+.nov {
+    background-color: #d3d3d3;
+}
+.lol {
+    height: 300px;
+    float: right;
+    margin-top: 100px;
+    padding-left: 50px;
+}
+.high {
+    height: 70px;
+    background-color: #7CE4E8
+}
+#avatar-size {
+    height: 60px;
+}
+.higho{
+    float: left;
+    height: 70px;
+    background-color:#7CE4E8    
+}
+.time {
+    font-size: 100px;
+    margin-top: 40px; 
+    background-color: black;
+    height: 300px;
+}
+.pad {
+    padding-left: 15px;
+    background-color: rgb(194, 202, 202)
+}
+.counter {
+    float:right;
+}
+.number-of-players {
+    float: left;
+
+}
+.invite {
+    background-color: black;
+    cursor:pointer;
+}
+.start-game{
+    cursor:pointer;
+}
+.rules {
+    background-color: black;
+}
+.how-to-play{
+    font-size: 30px;
+    text-align: center;
+    margin-bottom: 25px;
+}
+#center-0 {
+    margin: auto;
+    padding-left: 100px;
+}
+#time-remain {
+    font-size: 16px;
+    margin: auto;
+    padding-left: 70px;
+}
+.shift-right {
+    float: right;
+    margin-top: 45px; 
+}
+.text-name {
+    font-size: 30px;
+}
+.text-player{
+    font-size: 25px;
+}
+.lolo {
+    height: 300px;
+    margin-top: 55px;
+    /* float: right; */
+    
+}
+.head {
+    text-align: center;
+    color: black;
+}
+.position {
+    padding-top: 20%;
+    width: 60%;
+    margin: 0 auto;
+    text-align: center;
+}
+
+#name {
+    font-size: 60px;
+    padding-bottom: 20px;
+    -webkit-animation-delay: 1s;
+}
+
+#story {
+    padding-bottom: 20px;
+}
+
+.button1 {
+    padding-left: 0px 20px;
+    height: 60px;
+    background-color: transparent;
+    cursor: pointer;
+}
+.play {
+    margin: 0 auto;
+    padding: 20px 35px !important;
+}
+.back {
+    background-color: black;
+    color: white;
+}
+.donate {
+    border-color: white;
+    cursor:pointer;
+}
+#main-footer {
+    background-color: black;
+    padding: 30px 0px;
+}
+.info {
+    line-height: 200%;
+}
+.team {
+    border-radius: 50%;
+    width: 190px;
+    height: 190px;
+    margin: auto;
+}
+.team-name {
+    font-size: 16px;
+    padding-left: 30px;
+    padding-top: 10px;
+}
+.img-size {
+    width: 200px;
+    height: 250px;
+    display: block;
+    margin: 0 auto;    
+}
+
+.img-div {
+    margin: 0 auto;
+}
```

Comment:
Add new line
It might be easier to set up your editor to add it automatically when saving

Reply:
done